run.py

```diff
@@ -1,6 +1,14 @@
 from app.create_app import create_app
+from app.flask_config import Config
+from app.atlas_client import AtlasClient
+import pymongo
 
-app = create_app()
+app_config = Config()
+client = pymongo.MongoClient(f"{app_config.db_url}")
+db = client[f'{app_config.db_name}']
```

Comment:
As above, any particular reason to prefer this to `db = client[app_config.db_name]`?